docs/references/django_model_types.md

```diff
@@ -0,0 +1,49 @@
+# Django Model Types
+
+Django Models can be converted in Strawberry Types with the `@strawberry_django.type` decorator
+
+```python
+import strawberry
+
+@strawberry.django.type(models.Fruit)
+class Fruit:
+    ...
+```
+
+## Fields
+
+By default, no fields are implemented on the new type. For details on adding fields,
+see the [Fields](fields.md) documentation.
+
+## Queryset setup
+
+By default, a Strawberry Django type will pull data from the default manager for it's Django Model.
+You can change this behavior by implementing a `get_queryset` method on the type.
+
+```python
+@strawberry.django.type(models.Fruit)
+class Berry:
+
+    def get_queryset(self, queryset, info):
+        return queryset.filter(name__contains="berry")
+```
+
+The `get_queryset` method is given a Queryset to filter and
+a Strawberry `Info` object, containing details about the request.
+
+You can use the `info` parameter to limit access to items.
+
+```python
+@strawberry.django.type(models.Fruit)
+class Berry:
+
+    def get_queryset(self, queryset, info):
+        if info.context['request'].user.is_staff:
+            return queryset.filter(
+                name__contains="berry",
+            )
+        return queryset.filter(
+            name__contains="berry",
+            is_top_secret=False,
+        )
```

Comment:
Maybe rewrite this for clarity in the example:

```python
def get_queryset(self, queryset, info):
    if not info.context.request.user.is_staff:
        # Limit access to top secret if the user is not a staff member
        queryset = queryset.filter(
            is_top_secret=False,
        )

    return queryset.filter(
        name__contains="berry",
    )
```

Reply:
@bellini666 Will `info.context` has a property called `request`? I thought it was a dictionary that you would need to use square bracket lookups for

Reply:
@hiporox yes it does! You can actually access it both ways, by either doing `info.context.request` and `info.context["request"]`, but the attribute access has better static typing support in the current code (unless the one using it types the context themselves using a TypeDict).